M2Mqtt/Utility/Trace.cs

```diff
@@ -1,5 +1,6 @@
 ﻿/*
-Copyright (c) 2013, 2014 Paolo Patierno
+Copyright (c) .NET Foundation and Contributors
+Portions Copyright (c) 2013, 2014 Paolo Patierno
 
```

Comment:
would it make sens to have a trace class like this one for all the nano projects?